endpoints/currency/cb.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/first */
+const request = require('request')
+const h = require('apis-helpers')
+const cheerio = require('cheerio')
+const app = require('../../server')
+
+app.get('/currency/cb', (req, res) => {
+  request.get({
+    headers: { 'User-Agent': h.browser() },
+    url: 'https://www.sedlabanki.is/hagtolur/opinber-gengisskraning/',
+  }, (err, response, body) => {
+    if (err || response.statusCode !== 200) {
+      return res.status(500).json({ error: 'www.sedlabanki.is refuses to respond or give back data' })
+    }
+
+    const $ = cheerio.load(body)
+    const currencies = []
+
+    $('.Gengistafla').first().find('tr').each(function () {
+      const tds = $(this).find('td')
+      const name = tds.eq(0).text()
+
+      if (name) {
+        currencies.push({
+          shortName: name,
+          longName: tds.eq(1).text(),
+          value: tds.eq(4).text(),
```

Comment:
Looks like the html has changed. The table only has 3 columns now so the indexes in the `tds.eq` calls are wrong.

```suggestion
      const name = tds.eq(1).text()

      if (name) {
        currencies.push({
          shortName: name,
          longName: tds.eq(0).text().trim(),
          value: Number(tds.eq(2).text().replace(',', '.')),
```

I'm not sure if the `if` on line 23 is still valid now that `name` is from column index 1.